clients/admin-ui/src/features/common/table/v2/cells.tsx

```diff
@@ -6,10 +6,20 @@ import {
   Checkbox,
   CheckboxProps,
   Flex,
+  Switch,
   Text,
+  useDisclosure,
+  useToast,
+  WarningIcon,
 } from "@fidesui/react";
 import { HeaderContext } from "@tanstack/react-table";
-import { FC, ReactNode } from "react";
+import { ChangeEvent, FC, ReactNode } from "react";
+
+import { RTKResult } from "~/types/errors";
+
+import { getErrorMessage, isErrorResult } from "../../helpers";
+import ConfirmationModal from "../../modals/ConfirmationModal";
+import { errorToastParams } from "../../toast";
```

Comment:
We prefer absolute paths relative to the base directory `~` so we should use
```
../../helpers -> ~/features/common/helpers
../../modals/ConfirmationModal -> ~/features/common/modals/ConfirmationModal
../../toast -> ~/features/common/toast
```
This way the imports will always be accurate even if the components are refactored to be in a different location relative to each other